.renovaterc.json

```diff
@@ -4,85 +4,116 @@
     },
 
     {
       "paths": ["src/**"],
       "updateTypes": ["minor"],
-      "groupName": "runtime dependencies (minor updates)",
+      "groupName": "runtime dependencies",
       "reviewers": ["team:wg-runtime"],
+      "prBodyNotes": [
+        "I've added two members of `@ampproject/wg-runtime` to review this PR."
+      ],
       "labels": ["WG: runtime"]
     },
     {
       "paths": ["src/**"],
       "updateTypes": ["patch", "pin"],
-      "groupName": "runtime dependencies (patch updates)",
+      "groupName": "runtime dependencies",
       "reviewers": ["team:wg-runtime"],
+      "prBodyNotes": [
+        "I've added two members of `@ampproject/wg-runtime` to review this PR."
+      ],
       "labels": ["WG: runtime"]
     },
 
     {
       "paths": ["build-system/**"],
       "updateTypes": ["minor"],
-      "groupName": "build system dependencies (patch updates)",
+      "groupName": "build system dependencies",
       "reviewers": ["team:wg-infra"],
+      "prBodyNotes": [
+        "I've added two members of `@ampproject/wg-infra` to review this PR."
+      ],
       "labels": ["WG: infra"]
     },
     {
       "paths": ["build-system/**"],
       "updateTypes": ["patch", "pin"],
-      "groupName": "build system dependencies (minor updates)",
+      "groupName": "build system dependencies",
       "reviewers": ["team:wg-infra"],
+      "prBodyNotes": [
+        "I've added two members of `@ampproject/wg-infra` to review this PR."
+      ],
       "labels": ["WG: infra"]
     },
 
     {
       "paths": ["validator/**"],
       "updateTypes": ["minor"],
-      "groupName": "validator dependencies (patch updates)",
+      "groupName": "validator dependencies",
       "reviewers": ["team:wg-caching"],
+      "prBodyNotes": [
+        "I've added two members of `@ampproject/wg-caching` to review this PR."
+      ],
       "labels": ["WG: caching"]
     },
     {
       "paths": ["validator/**"],
       "updateTypes": ["patch", "pin"],
-      "groupName": "validator dependencies (minor updates)",
+      "groupName": "validator dependencies",
       "reviewers": ["team:wg-caching"],
+      "prBodyNotes": [
+        "I've added two members of `@ampproject/wg-caching` to review this PR."
+      ],
       "labels": ["WG: caching"]
     },
 
     {
       "paths": ["+(package.json)"],
       "updateTypes": ["minor"],
-      "groupName": "core dependencies (patch updates)"
+      "groupName": "core dependencies",
+      "prBodyNotes": ["/cc @ampproject/build-cop"]
     },
     {
       "paths": ["+(package.json)"],
       "updateTypes": ["patch", "pin"],
-      "groupName": "core dependencies (minor updates)"
+      "groupName": "core dependencies",
+      "prBodyNotes": ["/cc @ampproject/build-cop"]
+    },
+
+    {
+      "depTypeList": ["dependencies"],
+      "enabled": false
     },
 
     {
       "updateTypes": ["minor", "patch", "pin"],
       "packagePatterns": ["^@ampproject/"],
```

Comment:
Since this is a [regex](https://docs.renovatebot.com/configuration-options/#packagepatterns), does the forward slash need to be escaped? (`"^@ampproject\/"`)

Reply:
Examples I've seen say no, and `/` isn't a special character within the regex language itself, which is why it's frequently used as a quoting character a la `/this/that/s`

![image](https://user-images.githubusercontent.com/6694512/82266188-038c5180-9937-11ea-85e8-92db7a95fa77.png)